api/routes.go

```diff
@@ -49,6 +49,7 @@ func addRoutes(r *gin.Engine, auth Authentication, tokenHandler TokenHandler, uc
 	router.GET("/ast-expression/available-functions", tom, handleAvailableFunctions)
 
 	router.GET("/decisions", timeoutMiddleware(LIST_DECISION_TIMEOUT), handleListDecisions(uc, marbleAppHost))
+	router.GET("/decisions/internal", tom, handleListDecisionsInternal(uc, marbleAppHost))
```

Comment:
new: two different endpoints. This one should be used by the front and still returns the start/end indexes of the returned values.